test/espnet2/asr/test_pit_espnet_model.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import torch
+
+from espnet2.asr.ctc import CTC
+from espnet2.asr.decoder.transformer_decoder import TransformerDecoder
+from espnet2.asr.encoder.transformer_encoder_multispkr import TransformerEncoder
+from espnet2.asr.pit_espnet_model import ESPnetASRModel
+
+
+@pytest.mark.parametrize("encoder_arch", [TransformerEncoder])
+def test_pit_espnet_model(
+    encoder_arch,
+):
```

Comment:
```suggestion
def test_pit_espnet_model(encoder_arch):
```

Reply:
It seems the multi-speaker case (`num_inf > 1`) is not tested.

Reply:
will address it.